docs/contributor/release-testing.md

```diff
@@ -1,38 +1,43 @@
-# Documentation for Setting Up and Running the Release Testing GitHub Action
+# Set Up and Run the Release Testing GitHub Action
 
-This documentation provides a step-by-step guide to setting up and running the release testing GitHub Action for your backend's end-to-end (E2E) testing. Follow these steps to ensure your CI/CD pipeline is correctly configured and executed.
+This document provides a step-by-step guide to setting up and running the release testing GitHub Action for your backend's end-to-end (E2E) testing. Follow these steps to ensure your CI/CD pipeline is correctly configured and executed.
```

Comment:
This whole document is outdated as well. I'll create a new PR to document the testing we have so far.